scripts/python_tests.sh

```diff
@@ -23,5 +23,5 @@ echo "Superset config module: $SUPERSET_CONFIG"
 
 superset db upgrade
 superset init
-nosetests --stop tests/load_examples_test.py
-nosetests --stop --exclude=load_examples_test tests
+pytest --maxfail=1  tests/load_examples_test.py
```

Comment:
Do we want coverage for this file as well? I don't think it's needed but it was covered in previous setup.

Reply:
my preference is not to count it, to me this is more a fixture setup rather than an actual test.
However do not have a preference here.

Reply:
https://codecov.io/gh/apache/incubator-superset/pull/10177/commits - looks like it was the reason for dropped coverage - will keep playing with it